packages/typescript/template/README.md

```diff
@@ -0,0 +1,246 @@
+import Button from './Button'; // Import a component from another file
+
+const DangerButton = kind({
+  render(props) {
+    return <Button {...props} color="red" />;
+  }
+});
+
+export default DangerButton;
+```
+
+Be aware of the [difference between default and named exports](http://stackoverflow.com/questions/36795819/react-native-es-6-when-should-i-use-curly-braces-for-import/36796281#36796281). It is a common source of mistakes.
+
+We suggest that you stick to using default imports and exports when a module only exports a single thing (for example, a component). That’s what you get when you use `export default Button` and `import Button from './Button'`.
+
+Named exports are useful for utility modules that export several functions. A module may have at most one default export and as many named exports as you like.
+
+Learn more about ES6 modules:
+
+* [When to use the curly braces?](http://stackoverflow.com/questions/36795819/react-native-es-6-when-should-i-use-curly-braces-for-import/36796281#36796281)
+* [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
+* [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
+
+## Adding a LESS or CSS Stylesheet
+
+This project setup uses [Webpack](https://webpack.github.io/) for handling all assets. Webpack offers a custom way of “extending” the concept of `import` beyond JavaScript. To express that a JavaScript file depends on a LESS/CSS file, you need to **import the CSS from the JavaScript file**:
+
+### `Button.less`
+
+```css
+.button {
+  padding: 20px;
+}
+```
+
+### `Button.js`
+
+```js
+import kind from '@enact/core/kind';
+import styles './Button.css'; // Tell Webpack that Button.js uses these styles
+
+const Button = kind({
+  render() {
+    // You can use them as regular CSS styles
+    return <div className={styles['button']} />;
+  }
+}
+```
+
+Upon importing a css/less files, the resulting object will be a mapping of class names from that document. This allows correct access to the class name string regardless how the build process mangles it up.
+
+In development, expressing dependencies this way allows your styles to be reloaded on the fly as you edit them. In production, all CSS files will be concatenated into a single minified `.css` file in the build output.
+
+Additionally, this system setup supports [CSS module spec](https://github.com/css-modules/css-modules) with allows for compositional CSS classes and inheritance of styles.
+
+## Adding Images and Custom Fonts
+
+With Webpack, using static assets like images and fonts works similarly to CSS.
+
+You can **`import` an image right in a JavaScript module**. This tells Webpack to include that image in the bundle. Unlike CSS imports, importing an image or a font gives you a string value. This value is the final image path you can reference in your code.
+
+Here is an example:
+
+```js
+import kind from '@enact/core/kind';

```

Comment:
And here.

Reply:
That's just marking the markdown block for javascript syntax

Reply:
I realize that.  I meant that code should be updated to typescript standard.